tests/e2e/helpers/mobileSearch.js

```diff
@@ -0,0 +1,79 @@
+import { ClientFunction, RequestMock } from 'testcafe';
+import { SEARCH_INSIDE_URL_RE , mockResponseFound, mockResponseNotFound } from './mockSearch';
+import { TEST_TEXT_FOUND, TEST_TEXT_NOT_FOUND, PAGE_FIRST_RESULT } from './searchTestParams'
+
+
+export function runMobileSearchTests(br) {
+  //building mock response  for successful and unsuccessful search
+  const mockFound = RequestMock()
+    .onRequestTo(SEARCH_INSIDE_URL_RE )
+    .respond(mockResponseFound, 202);
+
+  const mockNotFound = RequestMock()
+    .onRequestTo(SEARCH_INSIDE_URL_RE )
+    .respond(mockResponseNotFound, 202);
+
+  test
+    .requestHooks(mockFound)('Mobile search - successful search', async t => {
+      await t.resizeWindowToFitDevice('Sony Xperia Z', {portraitOrientation: true})
+      const { nav } = br;
+
+      //opening side menu and search
+      await t.expect(nav.mobile.hamburgerButton.visible).ok();
+      await t.click(nav.mobile.hamburgerButton);
+      await t.expect(nav.mobile.menuSearchButton.visible).ok();
+      await t.click(nav.mobile.menuSearchButton);
+
+      //assuring that the search bar is enabled
+      await t.expect(nav.mobile.searchBox.visible).ok();
+
+      //testing successful search
+      await t
+        .selectText(nav.mobile.searchBox.child('.BRsearchInput'))
+        .pressKey('delete');
+      await t.typeText(nav.mobile.searchBox.child('.BRsearchInput'), TEST_TEXT_FOUND);
+      await t.click((nav.mobile.searchBox).child('.BRsearchSubmit'));
+      await t.expect(nav.mobile.searchResult.exists).ok();
+      await t.expect(nav.mobile.searchResultText.exists).ok();
+      await t.expect(nav.mobile.searchResultText.innerText).contains(TEST_TEXT_FOUND);
+
+      //checking url
+      const getPageUrl = ClientFunction(() => window.location.href);
+      await t.expect(getPageUrl()).contains(TEST_TEXT_FOUND);
+     
+      //checks clicking on first search result opens correct page
+      await t.click(nav.mobile.searchResult);
+      await t.expect(getPageUrl()).contains(PAGE_FIRST_RESULT);
+
```

Comment:
Let's also add a check that one of those blue annotation rectangles has been created.

Reply:
I have not quite understood what the annotation rectangles are... Do you mean the blue text highlight of the searched word?

Reply:
Yep! Those things :)

Reply:
Awesome! Thanks for also adding to the desktop tests!